backend/src/main/backend/Misc.txt

```diff
@@ -0,0 +1,32 @@
+Problems:
+--> [FIXED?]
+    the table class is too general. each instance of the class can create
+    tables for all the other tables.
+    For example, the Table object with name "tblMessages" can be used to 
+    create a new table in the database by a different name, "tblLikes."
+    * Need to make Table class more specific
+    * Store a list of Tables in Database.java?
+
+--> [IN PROGRESS]
+    App.java is not yet fully updated to be compatible with the new
+    Database.java
+    * each call to insert a row to a table will need to send the new row info
+      in a list. the order of this list will be sensitive (see insertRow())
+
+
+
+Other Notes:
+--> The Table object is different from a table created in the database. each
+    table in the database has a corresponding Table object that allows us to
+    perform operations on the table using generalized prepared statements.
+
+--> Some methods and operations have not yet been standardized:
```

Comment:
I think Mike hinted at the answer here - I recommend focusing less on pure standardization, and more on providing guardrails for the functionality where you can then write the semi-standardized code.